### PR TITLE
Updating MapSnapshotter.Options method list

### PR DESCRIPTION
**Pull request by langsmith**
This pr updates the chart in https://docs.mapbox.com/android/maps/overview/snapshotter/#taking-a-map-snapshot. Its list of `MapSnapshotter.Options` methods needed to be updated to match what's actually available to developers.

### CODE REVIEW

File: src/pages/maps/overview/snapshotter.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ Here are the various settings that are available within the `MapSnapshotter.Opti
 
 | Method | Description |
 | --- | --- |
+| `withApiBaseUrl` | Specifies the URL used for the Map API endpoint. It's very rare that you'll need to change the API base URL.
```
Comment (colleenmcginnis) [inline src/pages/maps/overview/snapshotter.md]: `Map API` or `Maps API`?

File: src/pages/maps/overview/clustering.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Showing clustered data is entirely possible by using the data-driven styling cap
 Adjusting the amount of data shown on the map to the map's camera zoom level, is a way to provide users with a cleaner UI experience and less overwhelming location data experience.
 
 
-## `CircleLayer`
+## CircleLayer
```
Comment (colleenmcginnis) [inline src/pages/maps/overview/clustering.md]: You need to keep this in back ticks `CircleLayer` or add "CircleLayer" to https://github.com/mapbox/android-docs/blob/publisher-production/android.dic to tell the spellcheck test to ignore it.

Comment (langsmith) [reply src/pages/maps/overview/clustering.md]: Adding to dictionary

File: src/pages/maps/overview/snapshotter.md (hunk 1 above)
Comment (langsmith) [reply src/pages/maps/overview/snapshotter.md]: Maps API

**Comment by langsmith** [comment]
@colleenmcginnis , pushed changes based on your review

**Comment by langsmith** [comment]
@colleenmcginnis 👈👈👈👈

**Pull request merged**